src/hazmat/primecount.rs

```diff
@@ -0,0 +1,117 @@
+use crypto_bigint::{Uint, U256};
+
+/// Estimate the number of primes smaller than x using the asymptotic expansion of Li(x) with three terms. 𝜋(𝑥)/Li(x)
+/// converges to 1 from below, which means that the estimation of number of primes for the ranges relevant here is a
+/// slight overestimate.
+/// Uses the formula 𝜋(𝑥) ~ (x/ln(x)) * (1 + 1!/ln(x) + 2!/(ln(x)^2) + 3!/(ln(x)^3)).
```

Comment:
Out of curiosity, did you compare this formula with other series representations, e.g. the Ramanujan's one?

Reply:
I compared  `x/ln x`, `x/(ln x +b * ln ln x)` (with b = -1, -0.4) and Riemann's R (as implemented in `num-prime`).

Reply:
Aha, no, I did not try the ramanjuan expansion. Interesting though, worth a go!

Reply:
I tried Ramanujan's series with 5 terms and it gives terrible results for known pi(x), much worse than the asymptotic series. 
From what I've read it looks like it needs way more terms (tens? hundreds?) to be reasonably accurate for the range 2^1024 - 2^8192 and is more suited for smaller x.